Makefile

```diff
@@ -117,9 +118,12 @@ update_base_url: \
 test: all
 	pytest src emsdk/tests packages/*/test* pyodide-build -v
 
+copy-fonts:
+	mkdir -p build/fonts
```

Comment:
This should also be moved to `matplotlib/meta.yaml` post section, above line 40.

Reply:
Done!